### PR TITLE
create PR

### PR DESCRIPTION
**Pull request by AlexKukhtinDev**

### CODE REVIEW

File: src/main/java/com/akukhtin/internetshop/model/Bucket.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package com.akukhtin.internetshop.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Bucket {
+    private static long idGenerator = 0;
```
Comment (Sofasmile) [inline src/main/java/com/akukhtin/internetshop/model/Bucket.java]: better move this logic(id generator) in the separate class

File: src/main/java/com/akukhtin/internetshop/service/BucketService.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+    Bucket update(Bucket bucket);
+
+    void delete(Long id);
+
+    Bucket addItem(Long bucketId, Long itemId);
+
+    Bucket clear(Long bucketId);
+
+    List getAllItems(Long bucketId);
```
Comment (Sofasmile) [inline src/main/java/com/akukhtin/internetshop/service/BucketService.java]:
```suggestion
    List<Item> getAllItems(Long bucketId);
```

File: src/main/java/com/akukhtin/internetshop/service/UserService.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+public interface UserService {
+    User create(User user);
+
+    User get(Long id);
+
+    User update(User user);
+
+    void delete(Long id);
+
+    List getOrders(Long userId);
```
Comment (Sofasmile) [inline src/main/java/com/akukhtin/internetshop/service/UserService.java]:
```suggestion
    List<Order> getOrders(Long userId);
```

Comment (AlexKukhtinDev) [reply src/main/java/com/akukhtin/internetshop/service/UserService.java]: OK

File: src/main/java/com/akukhtin/internetshop/model/Bucket.java (hunk 1 above)
Comment (AlexKukhtinDev) [reply src/main/java/com/akukhtin/internetshop/model/Bucket.java]: Why confuse? This static(private) field only this model

File: src/main/java/com/akukhtin/internetshop/service/BucketService.java (hunk 2 above)
Comment (AlexKukhtinDev) [reply src/main/java/com/akukhtin/internetshop/service/BucketService.java]: OK

**Pull request merged**